src/avgScrabbleScore.js

```diff
@@ -7,14 +7,25 @@ let scrabbleScore = require('./scrabbleScore');
  * Use your implementation of scrabbleScore.
  */
 function avgScrabbleScore(words) {
-  // This is your job. :)
+  let scores = 0;
+
+  for(let word of words) {
+    // console.log("score of " + word + " is " + scrabbleScore(word));
+    scores += scrabbleScore(word);
+    // console.log("scores is now " + scores);
+  }
+
+  /* -1 is there because the newline at the end of the sample-words.txt file adds an
+  empty element to the array, increasing length by 1 which changes
+  the average*/
+  return scores/(words.length-1);
```

Comment:
I'd assume the data is what the data is and make it the responsibility of the caller to pass good data. To handle the extra newline you can call `trimRight()` before passing in the data (like we discussed on Discord)